docs/QSG/docker.md

```diff
@@ -1,172 +1,87 @@
-# DAOS in Docker
+# QSG - DAOS in Docker
 
-This section describes how to build and deploy Docker images allowing to simulate a small cluster
-using DAOS as backend storage.  This small cluster is composed of the following three nodes:
+This section describes how to build and deploy Docker images to simulate a small cluster using DAOS as backend storage.  This small cluster is composed of the following three nodes:
 
 - The `daos-server` node running a DAOS server daemon managing data storage devices such as SCM or
   NVMe disks.
-- The `daos-admin` node allowing to manage the DAOS server thanks to `dmg`command.
-- The `daos-client` node using the the DAOS server to store data.
+- The `daos-admin` node is used to manage the DAOS server thanks to the `dmg`command.
+- The `daos-client` node uses the DAOS server to store data.
 
-At this time only emulated hardware storage are supported by this Docker platform:
+At this time only emulated hardware storage is supported by this Docker platform:
 
-- SCM (i.e. Storage Class Memory) are emulated with standard RAM memory.
+- SCM (i.e. Storage Class Memory) is emulated with standard RAM memory.
 - NVMe disks are emulated with a file device.
 
+!!!warning
+Virtual Docker networking is still in development. The following are known issues:
```

Comment:
Find this paragraph a little confusing. Virtual Docker networks are not under development. It is the support with DAOS of virtual network interface such as the Bridge which is not yet supported. The workaround is indeed to use one of the physical network interfaces other than the loopback interface.